Duplicati/Library/Main/Operation/Backup/UploadSyntheticFilelist.cs

```diff
@@ -111,6 +111,7 @@ orderby n.Key
                                 foreach (var p in options.ControlFiles.Split(new char[] {System.IO.Path.PathSeparator}, StringSplitOptions.RemoveEmptyEntries))
                                     fsw.AddControlFile(p, options.GetCompressionHintFromFilename(p));
 
+                            fsw.CreateFilesetFile(false);

```

Comment:
The comment of the commit with this change is useful to put in here as a comment too - if code is refactored that information is lost in the git history.

Reply:
Good point.  Revision 0f30604046b8f4eb8481118e42324128470bdaf2 adds a comment explaining the decision here.